docs/personalization/api_reference/tracking_api.md

```diff
@@ -336,6 +336,16 @@ can be provided in the whole shop.
 
 `GET https://event.perso.ibexa.co/api/[customerid]/basket/[userid]/[itemtypeid]/[itemid]`
 
+There are no query string parameters for this event.
+
+### Deletefrombasket event
+
+The Deletefrombasket is issued when the end user removes items from their shopping cart. 
+It could signify that the user has either lost interest in the product or already purchased it elsewhere. 
```

Comment:
>or already purchased
Better to mention here that we have special event BUY for it. To avoid user will send deletefrombasket event when user not removed item but bought it

Reply:
@dabrt 
The Deletefrombasket is issued when the end user removes items from their shopping cart. 
It could signify that the user has either lost interest in the product. 

This version of the sentence is good enough.